upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -321,6 +319,84 @@
                 </div>
               </div>
             </div>
+            <!-- RSS -->
+            <div id="section4" class="tab-pane fade in">
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-category"><?php echo $rss_category_title; ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_category" id="rss-category" class="form-control smaily-rss-options">
+                    <option value=''>All Products</option>
+                    <?php
+                    foreach ($rss_categories as $category) {
+                      $selected = $category['category_id'] === $rss_category ? 'selected' : '';
+                      echo("<option value=$category[category_id] $selected>$category[name]</option>");
+                    }
+                    ?>
+                    </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="col-sm-2 control-label" for="rss-sort-by"><?php echo $rss_sort_by_title; ?></label>
+                <div class="col-sm-10">
+                  <select name="smaily_for_opencart_rss_sort_by" id="rss-sort-by" class="form-control smaily-rss-options">
+                   <?php
+                    // All available options
+                    $sort_options = [
+                      'pd.name' => $sort_name,
+                      'p.model' => $sort_model,
+                      'p.price' => $sort_price,
+                      'p.status' => $sort_status,
+                      'p.sort_order' => $sort_order
+                    ];
```

Comment:
Provide this mapping in the controller.

Reply:
Added

Reply:
Yeah, views should contain as little logic as possible - they should be used only to display content and have minimal conditional logic. But let's not make these changes within this PR.